typescript/infra/hardhat.config.ts

```diff
@@ -83,13 +83,18 @@ task('kathy', 'Dispatches random abacus messages').setAction(
       const remoteId = ChainNameToDomainId[remote];
       const coreContracts = core.getContracts(local);
       const outbox = coreContracts.outbox.outbox;
+      const paymaster = coreContracts.interchainGasPaymaster;
       // Send a batch of messages to the destination chain to test
       // the relayer submitting only greedily
       for (let i = 0; i < 10; i++) {
-        await outbox.dispatch(
+        await recipient.dispatchToSelf(
```

Comment:
I think if we are going to have a dummy message sender/receiver we should name it as such?